test/functional/usability/usa-config-file.bats

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env bats
+
+# Author: Castulo Martinez
+# Email: castulo.martinez@intel.com
+
+load "../testlib"
+
+global_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -n test-bundle -f /file_1 "$TEST_NAME"
+
+}
+
+test_setup() {
+
+	create_config_file
+
+}
+
+test_teardown() {
+
+	destroy_config_file
+
+}
+
+global_teardown() {
+
+	destroy_test_environment "$TEST_NAME"
+
+}
+
+@test "USA010: Use option without section in the configuration file" {
```

Comment:
it may be slower than unit tests but it is really not slow, all 10 tests run in 2.554s, and functional tests are easier to create. What are the corner cases you think we are missing here?

Reply:
if it's easier to write, go for it.